csharp/extractor/Semmle.Extraction.Tests/Layout.cs

```diff
@@ -15,6 +15,8 @@ public void TestDefaultLayout()
             var layout = new Semmle.Extraction.Layout(null, null, null);
             var project = layout.LookupProjectOrNull("foo.cs");
 
+            Assert.NotNull(project);
```

Comment:
I am OK with using `!` in test code, but would it be better to do
```
if (project is null)
    throw new Xunit.Sdk.NullException(project);
```
and get rid of the `!`s?